src/components/policy/policy_regular/src/policy_manager_impl.cc

```diff
@@ -1322,6 +1337,9 @@ void PolicyManagerImpl::ResetRetrySequence(
   LOG4CXX_AUTO_TRACE(logger_);
   sync_primitives::AutoLock auto_lock(retry_sequence_lock_);
   retry_sequence_index_ = 0;
+  if (listener_->CanUpdate()) {
+    ptu_requested_ = false;
```

Comment:
When would this flag be reset if `CanUpdate` is false?

Reply:
@jacobkeeler this condition is no need here now I removed it in [b1acdb5](https://github.com/smartdevicelink/sdl_core/pull/3208/commits/b1acdb52e5f2764df74a72fae8692feec9c1ec79)